src/Quartz/Impl/AdoJobStore/SimplePropertiesTriggerPersistenceDelegateSupport.cs

```diff
@@ -110,17 +113,21 @@ public void Initialize(string tablePrefix, string schedName, IDbAccessor dbAcces
 
         protected string TablePrefix { get; private set; }
 
+        [Obsolete("Scheduler name is now added to queries as a parameter")]
         protected string SchedNameLiteral { get; private set; }
 
+        protected string SchedName { get; private set; }
```

Comment:
Anywhere the SchedNameLiteral property existed, I've added a SchedName so that anyone deriving from the class is able to get the property easily.